node.gyp

```diff
@@ -39,6 +38,7 @@
       'lib/_http_outgoing.js',
       'lib/_http_server.js',
       'lib/https.js',
+      'lib/_linklist.js',
```

Comment:
The list appeared to be in alphabetical order with `_linklist` appearing to be one of a small number of exceptions, so I moved it to the logical-seeming place in the list. Seemed like a good idea at the time, but happy to move it back if there is any objection to it whatsoever.